README.md

```diff
@@ -1,38 +1,45 @@
 # Roboto VF
 
-This is a variable version of Roboto intended to be a 1:1 match with the official non-variable release from Google.
+This is a variable version of Roboto, intended to be a 1:1 match with the official non-variable v2.136 release from [github.com/googlefonts/roboto/releases](https://github.com/googlefonts/roboto/releases).
 
-This is not an official Google project, but was enabled with generous funding by Google Fonts, who contracted Type Network.
-The Roboto family of instances contained 6 weights and two widths of normal, along with italic of the regular width.
-The project began by taking UFO instances generated during the build process of the Roboto v2.136 release, which have quadratic outlines. 
-The Thin, Regular and Bold UFOs required some fixes for interpolation compatibility, and a build script was written that preserves outline overlaps.
+This is not an official Google project, but was enabled with generous funding by Google Fonts, who contracted Type Network to make the Variable Fonts upgrade.
 
-* [/sources](sources/) contains new source UFOs
+* [/sources](sources/) contains source files: UFO fonts, designspace file, build script, TTX hinting source data
+* **[`/fonts/Roboto[ital,wdth,wght].ttf`](/fonts/Roboto[ital,wdth,wght].ttf) is the latest font**
 
-* [/fonts](fonts/) folder contains variation font TTFs
+## Development Process
 
-  * **Roboto[ital,wdth,wght].ttf** has deltas min, default, max, and also intermediate instances
+Roboto v2.136 has static fonts for 6 weights in roman and italic with a normal width, along with 3 weights in roman and italic with a condensed width.
 
-Both fonts have named instances for all the styles in the v2.136 release.
+This project began by taking UFO sources generated by the build system of that release, which have quadratic outlines, and fixing some interpolation compatibility issues in the Thin, Regular and Bold files.
+This work was done by [@asaumierdemers](https://github.com/asaumierdemers) and [@cjdunn](https://github.com/cjdunn) 
 
-## Install
+New sources for the condensed width that extends the weight range to match the normal width were made by [@sannorozco](https://github.com/sannorozco) using anisotropic interpolation and corrected by hand.
+With these, the TTF has named instances for all the styles in the v2.136 release, plus additional named instances for all styles along the Weight and Width axes given values in the OpenType OS/2 table.
+
+A new build script was written by [@m4rc1e](https://github.com/m4rc1e) that preserves outline overlaps and generates a Variable Font from these new UFOs; the earlier build system was thus discarded.
+The hinting was transplanted from the v2.136 release TTFs to VTT using 
```

Comment:
The hinting was never taken from v2.136. That mapper was simply written because the glyph order changed during Mike's hinting.

Reply:
Comments on hinting in the read me could include warnings that:
 to change contours by adding or deleting points, 
 or by removing overlaps, 
will require developers to rehint those glyphs. 

If review of something else, let me know.

Reply:
Yes, the mapper was made when we did the ‘diet’ version, that was discarded, so we preserved the full glyph repertoire and complete Mike’s hinting.